src/test/java/org/minions/devfund/danielmontecinos/moviedatabase/ActorTest.java

```diff
@@ -0,0 +1,73 @@
+package org.minions.devfund.danielmontecinos.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests Actor class.
+ */
+public class ActorTest {
+    private Actor actor;
+
+    /**
+     * Creates actor object.
+     */
+    @Before
+    public void setUp() {
+        actor = new Actor();
+    }
+
+    /**
+     * Tests testsGetNameMethod method.
+     */
+    @Test
+    public void testGetNameMethod() {
+        assertNull(actor.getName());
+    }
+
+    /**
+     * Tests testsSetNameMethod method.
+     */
+    @Test
+    public void testSetNameMethod() {
+        actor.setName("Actor 1");
+        assertEquals("Actor 1", actor.getName());
+    }
+
+    /**
+     * Tests getMovies method.
+     */
+    @Test
+    public void testGetMoviesMethod() {
+        assertTrue(actor.getMovies().isEmpty());
+    }
+
+    /**
+     * Tests setMovie method.
+     */
+    @Test
+    public void testSetMovieMethod() {
+        actor.setMovie("Movie 1");
+        actor.setMovie("Movie 2");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Movie 2" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)